tests/test-class-activitypub-activity-dispatcher.php

```diff
@@ -178,6 +178,65 @@ function ( $value, $type ) {
 		remove_filter( 'pre_http_request', array( $pre_http_request, 'filter' ), 10 );
 	}
 
+	public function test_dispatch_fallback_activity() {
+		$followers = array( 'https://example.com/author/jon' );
+
+		add_filter(
+			'activitypub_is_user_type_disabled',
+			function ( $disabled, $type ) {
+				return false;
+			},
+			10,
+			2
+		);
```

Comment:
```suggestion
		add_filter( 'activitypub_is_user_type_disabled', '__return_false' );
```
WordPress has a few of these "frequently used" functions to make that easier.

Reply:
Oh no, shame on me! I used these so many times in this PR, but changed this function so many times that I didn't even notice how it ended up 😂

Thanks!